TeachEasy/src/teacheasy/data/AudioData.java

```diff
@@ -0,0 +1,44 @@
+package teacheasy.data;
+
+/**
+ * A class to hold audio data
+ * @author sr896 sh1157
+ *
+ */
+public class AudioData extends PageObject {
+
+	boolean viewProgress;
+	String uri;
+	String url;
+	
+	/**Constructor Method*/
+	public AudioData (PageObjectType nType, float nXStart, float nYStart) {
+	

```

Comment:
Style: Potentially unnecessary white space? Pretty sure i've done this same thing in the PageObject() code however. The style guide recommends not having these blank lines unless an implementation comment is needed after the method name